core/src/main/java/org/hisp/dhis/android/core/mockwebserver/Dhis2MockServer.java

```diff
@@ -443,8 +432,4 @@ public String getBaseEndpoint() {
     public RecordedRequest takeRequest() throws InterruptedException {
         return server.takeRequest();
     }
-
-    public void addResponse(String method, String path, String responseName, int responseCode) {
```

Comment:
Do not remove this method, it is used in the app to mock responses. This class and this method are intentionally public.

Reply:
And what about the other methods removed in this script?

Reply:
The other methods are private, if they are not used the can be removed